src/input/plaintext.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace GOL\Input;
+
+use GOL\Boards\Board;
+use Ulrichsg\Getopt;
+
+/**
+ * Fills the board with a specific patter saved in the plaintext format.

```

Comment:
patter+n
It would also be good to document/explain the plaintext format so that the reader of the class knows how to write/create such a file.